qcodes/instrument_drivers/Spectrum/M4i.py

```diff
@@ -957,22 +1012,7 @@ def blockavg_hardware_trigger_acquisition(self, mV_range, nr_averages=10,
         self.general_command(pyspcm.M2CMD_CARD_START |
                              pyspcm.M2CMD_CARD_ENABLETRIGGER | pyspcm.M2CMD_CARD_WAITREADY)
 
-        # setup software buffer
-        sizeof32bit = 4
-        buffer_size = ct.c_int32 * memsize * numch
-        data_buffer = (buffer_size)()
-        data_pointer = ct.cast(data_buffer, ct.c_void_p)
-
-        # data acquisition
-        self._def_transfer64bit(
-            pyspcm.SPCM_BUF_DATA, pyspcm.SPCM_DIR_CARDTOPC, 0, data_pointer, 0, sizeof32bit * memsize * numch)
-        self.general_command(pyspcm.M2CMD_DATA_STARTDMA |
-                             pyspcm.M2CMD_DATA_WAITDMA)
-
-        # convert buffer to numpy array
-        data = ct.cast(data_pointer, ct.POINTER(buffer_size))
-        output = np.frombuffer(data.contents, dtype=ct.c_int32) / nr_averages
-        self._debug = output

```

Comment:
the same code should then also be removed from (line 866](https://github.com/QCoDeS/Qcodes/pull/1509/files#diff-28e84092864b9f07030db603c5e1f52eR844)